educator/src/Dscp/Educator/Web/Student/Error.hs

```diff
@@ -85,7 +85,6 @@ instance ToServantErr StudentAPIError where
         InvalidFormat{}          -> err400
         ServiceUnavailable{}     -> err503
         SomeDomainError err -> domainToServantErrNoReason err
-    toServantErr = toServantErrJustTag
```

Comment:
What's up with those changes? :thinking:

Reply:
I knew someone will ask.
The default implementation of this method serializes the error entirely and thus requires `JSON` instances of error.
Previously it was implemented with `toServantErrorJustTag`, which, as its name suggests, puts only the tag in the error. This implementation is suitable only when the API has no client because the client needs to know the error in details in order to deserialize it.
And recently Educator and Student APIs obtained their clients.